doc/api/util.md

```diff
@@ -55,6 +55,7 @@ added: v0.8.0
 The `util.deprecate()` method wraps the given `function` or class in such a way that
 it is marked as deprecated.
 
+<!-- eslint-disable prefer-rest-params -->
```

Comment:
If I get this right, in the version we are using (v1.0.0-beta.4), this comment relates only to the following block:

https://github.com/eslint/eslint-plugin-markdown#configuration-comments

There is an issue in v1.0.0-beta.6 though:

https://github.com/eslint/eslint-plugin-markdown/issues/69

But I hope it will be fixed soon, then we can safely upgrade.